entry.go

```diff
@@ -90,6 +92,46 @@ func (service *EntriesService) Get(spaceID, entryID string) (*Entry, error) {
 	return &entry, err
 }
 
+
+// Upsert updates or creates a new space
```

Comment:
The comment should be `// Upsert updates or creates a new Entry` no?

Reply:
ah yes, thanks for the catch!